tests/rocker_scripts/matrix.json

```diff
@@ -53,30 +52,12 @@
       "script_name": "install_quarto.sh",
       "script_arg": "prerelease"
     },
-    {
-      "base_image": "rocker/r-ver",
-      "tag": "4.0.5",
-      "script_name": "install_cuda-11.1.sh",
-      "script_arg": "none"
-    },
```

Comment:
Why?
If you don't want to test this, vote to remove the script itself.

Reply:
As noted in this pull request, I have removed all cuda tests from this matrix and put them into another file.  I think we should run the cuda scripts in a separate workflow, which as I said, I'd eventually like to also move to self-hosted runner that has an NVIDIA GPU because it seems like a good idea to test the cuda images on a cuda device and not just test that they build.  

I haven't added this to the cuda test yet because I am trying here to work in small steps that make some progress from our current issues.  I do think we might want to reconsider if cuda 11.1 is something we want to support long term, but but am not trying to deal with that issue here.

Reply:
I see, that makes sense. Thanks for the clarification.